src/services/utils.ts

```diff
@@ -60,59 +67,44 @@ const downloadFile = (blob: Blob, filename: string, type?: FileType) => {
     document.body.removeChild(link);
 };
 
-const prepareRequest = (init: any, token?: string) => {
+function prepareRequest(init?: InitRequest, token?: Token): RequestInit {
     if (!(typeof init === 'undefined' || typeof init === 'object')) {
         throw new TypeError(
-            'First argument of prepareRequest is not an object : ' + typeof init
+            `Argument 2 of backendFetch is not an object ${typeof init}`
```

Comment:
?

Reply:
just a bad copy from another project I think

Reply:
Nope it's the first argument, not the second

Reply:
> It's from the version of gridadmin, both are true in the end no ?

The message haven't been updated in gridadmin I remember.